### PR TITLE
feat(cli): seeder updates already existing entries

### PR DESCRIPTION
**Pull request by nnorbert**
Seeder updates already existing entries

### CODE REVIEW

File: packages/cli/src/commands/database/ogcio/ogcio-seeder-local.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
         },
         "organization_roles": [
             {
+                "id": "bb-public-servant",
```
Comment (alfonsograziano) [inline packages/cli/src/commands/database/ogcio/ogcio-seeder-local.json:24]: Do we have to update the deployed seeder as well?

Comment (nnorbert) [reply packages/cli/src/commands/database/ogcio/ogcio-seeder-local.json:24]: Oh, you are right!

Comment (nnorbert) [reply packages/cli/src/commands/database/ogcio/ogcio-seeder-local.json:24]: Done!

File: packages/core/src/libraries/ogcio-user.ts [hunk 2]
```diff
@@ -136,15 +136,10 @@ const getDomainFromEmail = (email: string): string | undefined => {
...
+  getRoles: (id: string) => Promise<Role>,
   insertUsersRoles: (usersRoles: CreateUsersRole[]) => Promise<QueryResult<QueryResultRow>>
 ) => {
-  const userRole = await getRoles('Citizen');
-
-  if (userRole === undefined) {
-    consoleLog.error(phrases.en.errors.role.default_role_missing);
-    return;
-  }
+  const userRole = await getRoles('bb_citizen');
```
Comment (peschina) [inline packages/core/src/libraries/ogcio-user.ts:142]: Shall we put `bb_citizen` and `bb-public-servant` into constants? Can we also stick to either snake case or kebab case? :)

**Review by peschina — APPROVED** [review]
LGTM, just a small comment

File: packages/core/src/libraries/ogcio-user.ts (hunk 2 above)
Comment (nnorbert) [reply packages/core/src/libraries/ogcio-user.ts:142]: It supposed to be kebab case, good catch. I will declare them into an enum.

Comment (nnorbert) [reply packages/core/src/libraries/ogcio-user.ts]: Done!

**Pull request merged**